core/src/main/java/hera/core/commands/Help.java

```diff
@@ -33,23 +33,33 @@
 		} else {
 			Command cmd = getHelp(commandName);
 			if (cmd == null) {
-				return MessageSender.send(HeraMsgSpec.getErrorSpec(channel)
-						.setDescription(String.format(HeraUtil.getLocalisation(LocalisationKey.ERROR_NOT_REAL_COMMAND, guild).getValue(), commandName))
-				).then();
+				return MessageHandler.send(channel, MessageSpec.getErrorSpec(messageSpec -> {
+					messageSpec.setDescription(String.format(HeraUtil.getLocalisation(LocalisationKey.ERROR_NOT_REAL_COMMAND, guild).getValue(), commandName));
+				})).then();
 			}
 			title = cmd.getName().toString().toLowerCase();
 			//this will break as soon as descriptions are localized
 			message = Mono.just(cmd.getDescription());
 		}
-		return message.flatMap(m -> MessageSender.send(HeraMsgSpec.getDefaultSpec(channel).setTitle(title).setDescription(m))).then();
+		return message.flatMap(m -> MessageHandler.send(channel, MessageSpec.getDefaultSpec(messageSpec -> {
+			messageSpec.setTitle(title);
+			messageSpec.setDescription(m);
+		}))).then();
 	}
 
 	private static Mono<String> getHelpFromCommandList(Mono<List<Command>> commands) {
 		return commands.flatMap(cmnds -> {
-			StringBuilder helpStringBuilder = new StringBuilder();
-			for (Command cmd : cmnds) {
-				helpStringBuilder.append(cmd.getName().toString().toLowerCase() + "\n");
-			}
+			final StringBuilder helpStringBuilder = new StringBuilder();
+
+			cmnds.stream()
+					.map(cmd -> cmd.getName().toString().toLowerCase())
+					.sorted()
+					.forEach(cmd -> {
+						helpStringBuilder.append("- ");
+						helpStringBuilder.append(cmd);
+						helpStringBuilder.append("\n");
+					});
+
```

Comment:
you think my code is not good enough >:(

Reply:
I just made it so its sorted alphabetically, which was easier to do using streams :)